controllers/ibmblockcsi_controller.go

```diff
@@ -541,6 +565,10 @@ func (r *IBMBlockCSIReconciler) isNodeReady(node *appsv1.DaemonSet) bool {
 	return node.Status.DesiredNumberScheduled == node.Status.NumberAvailable
 }
 
+func (r *IBMBlockCSIReconciler) isCallHomeReady(callHome *batchv1.CronJob) bool {
+	return len(callHome.Status.Active) == len(callHome.Status.Active)
```

Comment:
can this ever be false?
if so, when?
if not, please replace it with a logic that can distinguish between ready and not ready states

Reply:
@ArbelNathan: "removed"
@oriyarde: "thanks, resolving"